src/modules/add-task.js

```diff
@@ -0,0 +1,18 @@
+import displayDom from './displayDom.js';
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+const addTask = () => {
+  const form = document.querySelector('.task-form');
+  const taskInput = document.getElementById('task-input-field');
+
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    tasks.push({ description: taskInput.value, completed: false, index: tasks.length });
+    form.reset();
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    displayDom();
+  });
+};
```

Comment:
- [ ] Each task's index should represent its position in the array. Your tasks start at an index of 0, but they should start from 1. The first task should have an index of 1, the second one 2, the fifth one 5, and so on.

Reply:
Alright